src/Surfnet/Stepup/Identity/Identity.php

```diff
@@ -507,6 +513,19 @@ public function expressPreferredLocale(Locale $preferredLocale)
         $this->apply(new LocalePreferenceExpressedEvent($this->id, $this->institution, $preferredLocale));
     }
 
+    public function forget()
+    {
+        if ($this->forgotten) {
```

Comment:
The opposite should be added to the other API methods. Would consider `$this->assertNotForgotten()` or something